### ISSUE TITLE
tabbing/accessibility issue -- dg homepage

### ISSUE THREAD

**Issue by ToniBonittoGSA** [GSA/digitalgov.gov#1260]
When tabbing thru the new DG homepage, the highlighted outline is missing when one tabs to the six featured image spots (blogs and events) -- but is ok on the title links and other laces throughout the page.

**Comment by jeremyzilar** [comment]
That will be fixed in the new site on https://demo.digital.gov/ 👍
